### PR TITLE
[Release Guide] Document Release Cycles

### PR DESCRIPTION
**Pull request by mkolesnik**
Depends on #711 

Signed-off-by: Mike Kolesnik <mkolesni@redhat.com>

### CODE REVIEW

File: src/content/development/contribution-guide/_index.en.md [hunk 1]
```diff
@@ -46,7 +46,12 @@ To be expanded
 
 ## Release Cycles
 
-To be expanded
+The Submariner project follows time based release cycles, each cycle is 16 weeks long.
+Blocking bugs might cause a delay in the final release date, while new features will never be a cause fo delaying the release.
```
Comment (skitt) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
Blocking bugs might cause a delay in the final release date, while new features will never be a reason to delay the release.
```

**Review by skitt — DISMISSED** [review]

File: src/content/development/contribution-guide/_index.en.md [hunk 2]
```diff
@@ -46,7 +46,12 @@ To be expanded
 
 ## Release Cycles
 
-To be expanded
+The Submariner project follows time based release cycles, each cycle is 16 weeks long.
```
Comment (tpantelis) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
The Submariner project follows time based release cycles where each cycle is 16 weeks long.
```

File: src/content/development/contribution-guide/_index.en.md [hunk 3]
```diff
@@ -46,7 +46,12 @@ To be expanded
...
+Blocking bugs might cause a delay in the final release date, while new features will never be a reason to delay the release.
```
Comment (tpantelis) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
A blocking bug might cause a delay in the final release date however an incomplete new feature will not.
```

File: src/content/development/contribution-guide/_index.en.md [hunk 4]
```diff
@@ -46,7 +46,12 @@ To be expanded
...
+
+Features that were partially implemented in a given release will be considered "experimental" and will not have any support commitment.
```
Comment (tpantelis) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
A features that was partially implemented in a given release will be considered "experimental" and will not have any support commitment.
```

**Review by tpantelis — CHANGES_REQUESTED** [review]

File: src/content/development/contribution-guide/_index.en.md (hunk 3 above)
Comment (dfarrell07) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
Blocking bugs may delay final releases. New features shall not be a reason to delay releases.
```

File: src/content/development/contribution-guide/_index.en.md (hunk 2 above)
Comment (dfarrell07) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
Submariner follows 16 week long time-based release cycles.
```

File: src/content/development/contribution-guide/_index.en.md (hunk 4 above)
Comment (dfarrell07) [reply src/content/development/contribution-guide/_index.en.md]: I'd keep the original. I think it makes sense to be consistently plural here.

File: src/content/development/contribution-guide/_index.en.md (hunk 2 above)
Comment (mkolesnik) [reply src/content/development/contribution-guide/_index.en.md]: I think original is easier to understand, with this one it isnt clear what time-based relates to.
We want to emphasize that the cycles are time based hence I wrote it explicitly.

File: src/content/development/contribution-guide/_index.en.md (hunk 3 above)
Comment (mkolesnik) [reply src/content/development/contribution-guide/_index.en.md]: We're talking about the GA yes? So "releases" doesn't make sense in this context.

Also "shall not" sounds a bit too pompous IMHO :)

**Comment by mkolesnik** [comment]
Accidentally edited the previous "apply suggestions commit" instead of the new one, but regardless the suggestions were addressed

File: src/content/development/contribution-guide/_index.en.md [hunk 5]
```diff
@@ -46,7 +46,12 @@ To be expanded
 
 ## Release Cycles
 
-To be expanded
+The Submariner project follows time based release cycles where each cycle is 16 weeks long.
+While blocking bugs may delay the [general availability](#general-availability) release date, new features won't be a reason to delay it.
```
Comment (tpantelis) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
While blocking bugs may delay the [general availability](#general-availability) release date, new features will not.
```

**Review by tpantelis — CHANGES_REQUESTED** [review]

**Review by dfarrell07 — APPROVED** [review]

**Review by tpantelis — APPROVED** [review]

**Review by skitt — APPROVED** [review]

**Pull request merged**